analytics/migrations/0001_initial.py

```diff
@@ -1,4 +1,6 @@
-# Generated by Django 3.0.14 on 2021-11-17 22:12
```

Comment:
Don't change the existing migrations. When you change the database you need to add migrations.

Reply:
Added.